setup.py

```diff
@@ -294,7 +294,6 @@ def run_cythonize(force=False):
     setupkw["classifiers"] = [
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: BSD License',
```

Comment:
Why is this removed? The README says the code is available under a BSD license.

Reply:
Its a deprecated classifier.